cli/cortx-prvsnr-cli.spec

```diff
@@ -54,27 +54,71 @@ rm -rf %{buildroot}
 # TODO test
 # TODO IMPROVE current workaround is to prevent conflicts
 #              with provisioner main rpm instllation
-cp -fpr /opt/seagate/cortx/provisioner/cli/src/* /opt/seagate/cortx/provisioner/cli
-chmod -R 750 /opt/seagate/cortx/provisioner/cli
+/bin/cat <<EOL > ssh_config
+Host srvnode-1 srvnode-1.colo.seagate.com
+    HostName srvnode-1.colo.seagate.com
+    User root
+    UserKnownHostsFile /dev/null
+    StrictHostKeyChecking no
+    IdentityFile /root/.ssh/id_rsa_prvsnr
+    IdentitiesOnly yes
+    LogLevel ERROR
+    BatchMode yes
+
+Host srvnode-2 srvnode-2.colo.seagate.com
+    HostName srvnode-2.colo.seagate.com
+    User root
+    UserKnownHostsFile /dev/null
+    StrictHostKeyChecking no
+    IdentityFile /root/.ssh/id_rsa_prvsnr
+    IdentitiesOnly yes
+    LogLevel ERROR
+    BatchMode yes
+EOL
+
+#/bin/ssh-keygen -o -q -t rsa -b 4096 -a 100 -N '' -f id_rsa_prvsnr
+#/bin/mv id_rsa_prvsnr* /opt/seagate/cortx/provisioner/files/.ssh
+#/bin/mv ssh_config /opt/seagate/cortx/provisioner/files/.ssh
+#/bin/cp -fpr /opt/seagate/cortx/provisioner/cli/src/* /opt/seagate/cortx/provisioner/cli
+#/bin/chmod -R 750 /opt/seagate/cortx/provisioner/cli
 
 # TODO test
-mkdir -p /root/.ssh
+/bin/mkdir -p /root/.ssh
 
-# Ensure update replaces the keys
-if [[ -e /root/.ssh/id_rsa_prvsnr ]]; then
-  rm -f /root/.ssh/id_rsa_prvsnr
-  rm -f /root/.ssh/id_rsa_prvsnr.pub || true
+## Ensure update replaces the keys
+#if [[ -e /root/.ssh/id_rsa_prvsnr ]]; then
+#  /bin/rm -f /root/.ssh/id_rsa_prvsnr || true
+#  /bin/rm -f /root/.ssh/id_rsa_prvsnr.pub || true
+#fi
+
+
+if [[ ! -e /root/.ssh/ssh_config || ! -e /root/.ssh/authorized_keys 
+  || ! -e /root/.ssh/id_rsa_prvsnr.pub || ! -e /root/.ssh/id_rsa_prvsnr ]]; then
```

Comment:
This won't work.
We are checking all or none. We should just check `-e /root/.ssh/id_rsa_prvsnr`
ssh_config should be tested separately for it's presense.
authorized_keys shouldn't be checked.